src/libraries/System.Runtime/tests/System/Reflection/NullabilityInfoContextTests.cs

```diff
@@ -1298,6 +1463,17 @@ public class TypeWithNotNullContext
         public void MethodNonNullNonNullNotNull(string s, [NotNull] IDictionary<Type, string[]?>? dict) { dict = new Dictionary<Type, string[]?>(); }
         public void MethodNullNonNullNullNon(string? s, IDictionary<Type, string?[]?> dict) { }
         public void MethodAllowNullNonNonNonNull([AllowNull] string s, IDictionary<Type, string[]>? dict) { }
+
+        public ref readonly int? MethodWithByRefs(out int? a, out string b, ref object? c, ref Type d, in decimal? e, in ICloneable f) =>
+            throw new NotImplementedException();
+        public ref Tuple<int?, string, IDisposable?> MethodWithGenericByRefs(
+            out KeyValuePair<string?, object> a,
+            ref Tuple<Type?, decimal>? b,
+            in KeyValuePair<ICloneable, IFormatProvider?>? c) =>
+            throw new NotImplementedException();
+        public ref T ConstrainedGenericMethodWithByRef<T>(out T[] array) where T : class? => throw new NotImplementedException();
+        public unsafe void MethodWithPointers(int* a, int?* b) { }
+        public unsafe void GenericMethodWithPointers<T>(T* a, T?* b) where T : unmanaged { }
```

Comment:
The issue was only about ByRef types, but I thought it made sense to add coverage for pointer types as well since those get treated very similarly.